### DISCUSSION TITLE
24/7,365 mozilla hubs AWS Server woes

### DISCUSSION THREAD

**Discussion by YannickBoldly** [mozilla/hubs#3645] [Hubs Cloud Questions] [answered]
Hi Guys!

For one of our clients, we are currently planning to launch a mozilla hubs meeting environment that is supposed to run 24/7, 365. 
In our initial calculations, it is safe to say we were flabbergasted by the costs that would incur. 
On aurora serverless alone we'd spend 365x24x0,24= 1,051 dollars a year. 
Let alone the rest of the costs that will be added to that. 

It wouldn't be this bad if we could display "hey! seems like you are the first one here, let us set the room for you"
But the spin-up for the server is an incredibly sluggish 1-3 minutes, which is frankly way to long for our usecase. 

I was wondering, do you guys have any succes stories on running hubs 24/7 without breaking the bank? 
Is there a cheaper solution than aurora?
Can we programmatically turn the server on or off on a client facing webpage?
Any other tips or tricks to help us would be absolutely appreciated. 

<3

**Comment by robinkwilson** [comment, marked as answer]
I always have database pausing on for all my instances to stop the serverless db costs. Is that the 1-3 minutes you're talking about?
It only spends that time spinning up when you first connect, subsequent connects should be instantaneous. -- I want to make sure we're not talking about going from Offline mode to Online mode.

We have one event we know about: VRTO (https://conference.virtualreality.to/) VRTO had a Hubs Cloud instance up for 1 month 24/7. I recommend reading any articles around that conference in Hubs.

**Reply by YannickBoldly** [reply]
Hi Robin!

Thanks for your reply!  (and your amazing tutorial videos)

We're talking about pausing, yes. 
We would pitch this as a true virtual extension of their workplace, so the drop in, drop out functionality is something that really aides that.
The database pausing requires quite a long time for the first visitor to get in, which greatly reduces the feeling of "snappiness" of the system. 
The apart poster gallery, which is live with many rooms at the same time, must churn a lot of AWS credits as well I think? 

We really, really love mozilla hubs, but the running costs are still something we're trying to make a system for. 

Because it also complexes quoting these projects for our clients. We can accurately scope creating bespoke environments or functionality, but we would love to be able to scope monthly costs better and more accessible to smaller clients.
